about/strategy.md

```diff
@@ -0,0 +1,102 @@
+- Microsoft Azure
+
+### Why Jupyter and JupyterHub?
+
+- The Jupyter ecosystem is a collection of building blocks that are highly customizable and composable. They are popular and useful for many use-cases, but still require expertise to customize for a particular need. This is well-suited for 2i2c's skillset and the kind of service it wishes to provide.
+- Jupyter is a community-led and multi-stakeholder ecosystem that aligns well with 2i2c's commitment to vendor-agnosticity and the Right to Replicate
+- JupyterHub allows you to access centralized infrastructure for a community, but in a way that gives that community a lot of control over the details. It is a good balance between "SaaS" and "Fully bespoke community infrastructure". JupyterHub can be deployed via a single repository, but is also deployable by individual people or communities, providing them a clear off-ramp.
+
+
+## Major questions to answer
+
+Below are the major questions we'd like to answer with this pilot.
+
+- How much work does it take to manage a single JupyterHub?
+- What are the major opportunities to improve technology or process to reduce this number?
+- What is the balance of work between development, operations, administration, and sales?
+- What are the major use-cases that can be met with repeatable JupyterHub distributions?
+- What kind of support model is sustainable for our team?
+- What are the major roles that should exist for a given hub? (both on the 2i2c side and the Community side)
+- What other services do communities want other than just a JupyterHub? How would the JupyterHub connect with other services?
+- What new development is needed to facilitate collaboration, communication, and exploration on a JupyterHub?
+
+## Deployment and operations strategy
+
+Our goal is to provide a service that minimizes infrastructural complexity while providing JupyterHubs that can be used my Communities of Practice independently of one another.
+We wish to minimize the amount of engineering labor needed to develop and operate these deployments.
+Below are a few major aspects of the service that we believe provide a good chance of accomplishing these goals:
+
+- Deploy independent JupyterHubs from a centralized deployment system
+- Use Terraform to build Kubernetes clusters on cloud providers, and Kubernetes as a base to run the actual JupyterHub infrastructure.
+- JupyterHubs should be pre-configured for a use-case, but customizable by the community
+- JupyterHubs will respect the Community's Right to Replicate
+- JupyterHubs may be more bespoke than is sustainable provided we can learn from them.
+- JupyterHubs should be able to connect with external datasets and services, as teh community needs.
+- JupyterHubs should be customizable by the communities they serve, ideally without intervention from a 2i2c Engineer.
+
+## Our Timeframe
+- Begin serving hub infrastructure immediately, as long as we do not over-extend our team
+- Finish one iteration in 6-months
```

Comment:
Let's put a date on this? I'm not sure what the start time is.

Reply:
good point - let's put Q1 2022 as a target date?